h/static/scripts/directive/annotation.coffee

```diff
@@ -61,19 +61,24 @@ AnnotationController = [
 
     model = $scope.annotationGet()
 
+    # Set model.permissions to the default permissions for new annotations,
+    # _if_ model.permissions isn't already set.
+    setDefaultPermissions = ->
+      if not model.permissions?.read
```

Comment:
Previously it was broken, and this was the reason why.

I'll dig into it further